Ballerina.lock

```diff
@@ -1,4 +1,344 @@
 org_name = "ballerinax"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
the version number should be 0.99.0 for the preview release

Reply:
Changed.